xdsl/interactive/app.py

```diff
@@ -52,6 +67,35 @@ def condensed_pass_list(input: builtin.ModuleOp) -> tuple[type[ModulePass], ...]
     return selections
 
 
+class LoadFile(Screen[str]):
```

Comment:
This screen seems like a good piece of code to factor out into its own file, as well as its corresponding tcss file.